scqubits/core/hilbert_space.py

```diff
@@ -923,6 +923,74 @@ def get_spectrum_vs_paramvals(
             state_table=eigenstate_table,
         )
 
+    def standardize_eigenvector_phases(self) -> None:
```

Comment:
There is currently a function `standardize_phases` in utils.spectrum_utils.py. This method seems very related (though it does the job for a different type of data structure) Perhaps we should turn it into a function and shift it to spectrum_utils.py as well.